Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -3,7 +3,7 @@
 from requests import Response
 from py42.sdk import SDKClient
 from py42.response import Py42Response
-from Code42 import (
+from .Code42 import (
```

Comment:
Shouldn't have to do this dot import, I don't see other packs doing this.

Reply:
this makes my intellisense work better, but makes the tests fail.